app/controllers/exchange_rates_controller.rb

```diff
@@ -0,0 +1,32 @@
+class ExchangeRatesController < ApplicationController
+  layout "application"
+  
+  def index
+    # Obtener monedas disponibles para el selector
+    @currencies = Currency.active.order(:code)
+    
+    # Configurar valores predeterminados
+    @from_currency = params[:from_currency] || Currency.default.code || "USD"
+    @to_currency = params[:to_currency] || (@currencies.where.not(code: @from_currency).first&.code || "EUR")
```

Comment:
Grouping unneeded
```suggestion
    @to_currency = params[:to_currency] || @currencies.where.not(code: @from_currency).first&.code || "EUR"
```
Also this line is too complicated and hard to read; extract to explaining var improves readability:
```suggestion
    first_other_currency = @currencies.where.not(code: @from_currency).first&.code
    @to_currency = params[:to_currency] || first_other_currency || "EUR"
```